.vscode/launch.json

```diff
@@ -213,18 +213,18 @@
             "request": "launch",
             "runtimeExecutable": "${execPath}",
             "args": [
-                "${workspaceFolder}/test/extension-tests/vs-preferred-gen/project-folder",
+                "${workspaceFolder}/test/extension-tests/successful-build/project-folder",
```

Comment:
Right. The original path was not working, there's no vs-preferred-gen in the CMake Tools test repo. But indeed the above change is redundant. So I'd rather delete the whole block instead than revert because it wasn't doing anything before.